### PR TITLE
Add taar_whitelist to the taar_amodump DAG

### PR DESCRIPTION
**Pull request by crankycoder**
This schedules the TAAR AMO whitelist ETL job.  This job is strictly dependant on `taar_amodump` completing.

### CODE REVIEW

**Comment by mlopatka** [comment]
@harterrt are you the right one to ask about getting this reviewed and merged in?

**Comment by harterrt** [comment]
Looks like @acmiyaguchi is on it. Thanks!

File: dags/taar_amodump.py [hunk 1]
```diff
@@ -33,3 +33,21 @@
     output_visibility="private",
     dag=dag
 )
+
+amowhitelist = EMRSparkOperator(
+    task_id="taar_amowhitelist",
+    job_name="Transform the AMO addon JSON into a whitelisted set of addons for futher TAAR processing",
```
Comment (acmiyaguchi) [inline dags/taar_amodump.py]: This job name should be short, its the identifier in the EMR console. Maybe something like "Update TAAR AMO-whitelist"

Comment (crankycoder) [reply dags/taar_amodump.py]: Thanks, I've shortened the job_name now.

**Comment by crankycoder** [comment]
make run COMMAND="test taar_amodump taar_amowhitelist 20180401" successfully spins up the the taar_amowhitelist job for me on my local docker setup.

```
$ make run COMMAND="test taar_amodump taar_amowhitelist 20180401"
docker-compose run -e AWS_SECRET_ACCESS_KEY -e AWS_ACCESS_KEY_ID web airflow test taar_amodump taar_amowhitelist 20180401
Starting telemetryairflow_db_1    ... done
Starting telemetryairflow_app_1   ... done
Starting telemetryairflow_redis_1 ... done
Waiting for db to listen on 5432...
Waiting for redis to listen on 6379...
/usr/local/lib/python2.7/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
  """)
[2018-04-09 14:16:41,204] {__init__.py:57} INFO - Using executor CeleryExecutor
/usr/local/lib/python2.7/site-packages/airflow/www/app.py:23: FlaskWTFDeprecationWarning: "flask_wtf.CsrfProtect" has been renamed to "CSRFProtect" and will be removed in 1.0.
  csrf = CsrfProtect()
[2018-04-09 14:16:41,811] {models.py:168} INFO - Filling up the DagBag from /app/dags
/usr/local/lib/python2.7/site-packages/airflow/utils/helpers.py:406: DeprecationWarning: Importing ExternalTaskSensor directly from <module 'airflow.operators' from '/usr/local/lib/python2.7/site-packages/airflow/operators/__init__.pyc'> has been deprecated. Please import from '<module 'airflow.operators' from '/usr/local/lib/python2.7/site-packages/airflow/operators/__init__.pyc'>.[operator_module]' instead. Support for direct imports will be dropped entirely in Airflow 2.0.
  DeprecationWarning)
/usr/local/lib/python2.7/site-packages/airflow/utils/helpers.py:406: DeprecationWarning: Importing BashOperator directly from <module 'airflow.operators' from '/usr/local/lib/python2.7/site-packages/airflow/operators/__init__.pyc'> has been deprecated. Please import from '<module 'airflow.operators' from '/usr/local/lib/python2.7/site-packages/airflow/operators/__init__.pyc'>.[operator_module]' instead. Support for direct imports will be dropped entirely in Airflow 2.0.
  DeprecationWarning)
/usr/local/lib/python2.7/site-packages/airflow/utils/helpers.py:406: DeprecationWarning: Importing BaseHook directly from <module 'airflow.hooks' from '/usr/local/lib/python2.7/site-packages/airflow/hooks/__init__.pyc'> has been deprecated. Please import from '<module 'airflow.hooks' from '/usr/local/lib/python2.7/site-packages/airflow/hooks/__init__.pyc'>.[operator_module]' instead. Support for direct imports will be dropped entirely in Airflow 2.0.
  DeprecationWarning)
[2018-04-09 14:16:43,927] {models.py:1128} INFO - Dependencies all met for <TaskInstance: taar_amodump.taar_amowhitelist 2018-04-01 00:00:00 [None]>
[2018-04-09 14:16:43,930] {models.py:1128} INFO - Dependencies all met for <TaskInstance: taar_amodump.taar_amowhitelist 2018-04-01 00:00:00 [None]>
[2018-04-09 14:16:43,931] {models.py:1334} INFO -
--------------------------------------------------------------------------------
Starting attempt 1 of 4
--------------------------------------------------------------------------------

[2018-04-09 14:16:43,935] {models.py:1358} INFO - Executing <Task(EMRSparkOperator): taar_amowhitelist> on 2018-04-01 00:00:00
[2018-04-09 14:16:43,956] {credentials.py:910} INFO - Found credentials in environment variables.
[2018-04-09 14:16:44,845] {connectionpool.py:735} INFO - Starting new HTTPS connection (1): us-west-2.elasticmapreduce.amazonaws.com
[2018-04-09 14:16:45,928] {emr_spark_operator.py:171} INFO - Running Spark Job Generate a whitelisted set of addons for TAAR with JobFlow ID j-NN4Z6DCCLPFP
[2018-04-09 14:16:45,928] {emr_spark_operator.py:182} INFO - Logs will be available at: https://console.aws.amazon.com/s3/home?region=us-west-2#&bucket=telemetry-test-bucket&prefix=logs/vng@mozilla.com/Generate a whitelisted set of addons for TAAR/j-NN4Z6DCCLPFP
[2018-04-09 14:16:46,159] {emr_spark_operator.py:210} INFO - Spark Job 'Generate a whitelisted set of addons for TAAR' status' is STARTING```

**Pull request merged**